Try fix multiple definitions of symbols error in elf on sh4

--- a/elf/Makefile	2016-01-05 19:45:43.451536892 +0200
+++ b/elf/Makefile	2016-01-05 19:48:10.256264853 +0200
@@ -48,6 +48,7 @@
 rtld-routines	= rtld $(all-dl-routines) dl-sysdep dl-environ dl-minimal
 all-rtld-routines = $(rtld-routines) $(sysdep-rtld-routines)
 
+CFLAGS-dl-conflict.c = -fno-delete-null-pointer-checks
 CFLAGS-dl-runtime.c = -fexceptions -fasynchronous-unwind-tables
 CFLAGS-dl-lookup.c = -fexceptions -fasynchronous-unwind-tables
 CFLAGS-dl-iterate-phdr.c = $(uses-callbacks)
